release-notes/index.dita

```diff
@@ -24,6 +25,86 @@
+          </li>
+          <li id="3680">In DITA-OT 3.6, branch filtering with nested subtopics failed with an IllegalArgumentException:
+              <msgph>URI has a fragment component</msgph>. Processing has been updated to correctly transform references
+            that contain anchors.
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3680" format="html" scope="external">#3680</xref>,
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3696" format="html" scope="external">#3696</xref>
+          </li>
+          <li id="3687">DITA-OT 3.6 in-memory processing calculated absolute paths incorrectly when merging maps. The
+            merge map and topic parsers have been updated to properly compute absolute locations.
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3687" format="html" scope="external">#3687</xref>,
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3699" format="html" scope="external">#3699</xref>
+          </li>
+          <li id="3688">DITA-OT 3.6 introduced a regression that prevented PDF2 custom font mappings from resolving
+            properly. The PDF pre-processor task has been updated with a delegating URI resolver to ensure that custom
+            resources in PDF plug-ins are applied as expected.
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3688" format="html" scope="external">#3688</xref>,
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3695" format="html" scope="external">#3695</xref>
+          </li>
+          <li id="3689">Custom rules used to adjust the names and locations of output files in the map-first
+            pre-processing routine were no longer applied in DITA-OT 3.6. Preprocessing has been updated to ensure that
+            stylesheets specified via the <parmname>result.rewrite-rule.xsl</parmname> parameter are correctly applied.
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3689" format="html" scope="external">#3689</xref>,
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3697" format="html" scope="external">#3697</xref>
+          </li>
+          <li id="3694">In DITA-OT 3.6, a regression prevented multi-level key references from resolving properly.
+            Processing has been updated to ensure that topic metadata is correctly copied between key definitions and
+            key references that point to other keys.
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3694" format="html" scope="external">#3694</xref>,
+            <xref href="https://github.com/dita-ot/dita-ot/issues/3701" format="html" scope="external">#3701</xref>
+          </li>
+        </ul>
+        <!-- TODO: Describe Docker container changes & docs updates -->
+        <!-- <p>The documentation for DITA Open Toolkit Release 3.1 provides corrections and improvements to existing topics,
```

Comment:
Are the comments still needed?

Reply:
Yes, that `TODO` is still open. 

When the initial draft was prepared, a related PR was in progress, so the necessary keys weren't yet available on the hotfix branch. Now that #334 has landed, I can add that info and remove the comment. _On that now…_